LeetCode/linkedlists.md

```diff
@@ -384,10 +384,6 @@ https://leetcode.com/problems/sort-list/
 #         self.next = next
```

Comment:
Your Pull Request title: LEETCODE Readded linkedlists.md

Check result:

* Pull Request title must contain group number in ['1021', '1022']
* Pull Request title action must start with ['Added', 'Deleted', 'Refactored', 'Moved', 'Fixed']
* Pull Request title must contain group number in ['1021', '1022']
* Pull Request title action must start with ['Added', 'Deleted', 'Refactored', 'Moved', 'Fixed']